packages/govern-console/src/environment/types.ts

```diff
@@ -0,0 +1,23 @@
+import { Bytes } from 'ethers';
+export type EnvironmentName = 'mainnet' | 'rinkeby' | 'staging';
+type ChainId = 1 | 4;
```

Comment:
there is a chainId enum type at: https://github.com/aragon/govern/blob/feature/dao-392-console-environment/packages/govern-console/src/utils/types.tsx#L62

Do you think having this 2 might cause confusion down the road?

Reply:
Good catch! I'll remove the enum ChainId as it's not used anywhere anymore.